gimel/gimel.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import print_function
 import sys
 sys.path.insert(0, './vendor')
-import redis
-from config import config
+import redis  # noqa
+from config import config  # noqa
```

Comment:
why do these imports need to be excluded from travis? (I'm assuming `# noqa` does that)